src/main/java/org/jenkinsci/test/acceptance/plugins/git/GitScm.java

```diff
@@ -155,6 +186,24 @@ public SparseCheckoutPaths sparseCheckout() {
     }
 
     /**
+     * Add behaviour "Advanced clone behaviours"
+     *
+     * @return behaviour, to access its method
+     */
+    public AdvancedClone advancedClone() {
+        return addBehaviour(AdvancedClone.class);
+    }
+
+    /**
+     * Add behaviour "Advanced checkout behaviours"
+     *
+     * @return behaviour, to access its method
+     */
+    public AdvancedCheckout advancedCheckout() {
+        return addBehaviour(AdvancedCheckout.class);
```

Comment:
For these ones, you don't want to return this?

Reply:
My intention is allow to configure the "Advanced checkout" behaviour any time you need in your test to compare the different results.